packages/block-library/src/post-template/edit.js

```diff
@@ -149,7 +154,7 @@ export default function PostTemplateEdit( {
 					>
 						{ blockContext ===
 						( activeBlockContext || blockContexts[ 0 ] ) ? (
-							<li { ...innerBlocksProps } />
+							<PostTemplateInnerBlocks />
```

Comment:
I think it makes a lot of sense because in practice this creates a list of Post Template inner blocks so all block props should be computed in that context that is set by the `BlockContextProvider`.

@michalczaplinski, we will need to apply the same changes as in the final version of this PR to the Comment Template block.

Reply:
Opened https://github.com/WordPress/gutenberg/pull/36174 to align the Comment Template block.

Reply:
Whoop, sorry for getting to this late @gziolo and thanks for taking care of this! 😊